### PR TITLE
Helm Chart Releaser Trigger Fix

### PR DESCRIPTION
**Pull request by mprimeaux**
### Description
Increment the version of each chart so the Helm chart releaser detects a difference as compared to previous executions.
 
### Issues Resolved
This resolves issue #55. 
 
### Check List
- [ ✔️ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by mprimeaux** [comment]
@peterzhuamazon Here are various proof points of testing, linting, and releasing working on my forked repository and testing and linting working in the OpenSearch Project repository.

* Linting and Testing on my forked repository is [here](https://github.com/opensearch-project/helm-charts/runs/3739594623?check_suite_focus=true).
* Releasing on my forked repository is [here](https://github.com/mprimeaux/helm-charts/runs/3739627913?check_suite_focus=true). The `helm` index is [here](https://github.com/mprimeaux/helm-charts/blob/gh-pages/index.yaml).
* Linting and Testing on this repository is [here](https://github.com/opensearch-project/helm-charts/runs/3739594623?check_suite_focus=true).

Once this PR is merged, I expect releasing will work on this repository. I am available day and night for questions or clarifications. If required, I am happy to coordinate an active pairing session with you and any of the maintainers to review this in real-time.

Proof of `helm repo add` working on my forked repository:

```shell
❯ helm repo add opensearch https://mprimeaux.github.io/helm-charts/
"opensearch" has been added to your repositories

❯ helm search repo  opensearch
NAME                              	CHART VERSION	APP VERSION	DESCRIPTION
opensearch/opensearch             	1.0.3        	1.0.0      	A Helm chart for OpenSearch
opensearch/opensearch-dashboards  	1.0.3        	1.0.0      	A Helm chart for OpenSearch Dashboards
```

Again, thanks much for your continued support. We got this.

**Comment by mprimeaux** [comment]
@DandyDeveloper @TheAlgo @smlx As always, I would sincerely appreciate your comments and review.

**Review by DandyDeveloper — APPROVED** [review]
I see nothing wrong with these changes. LGTM

**Review by peterzhuamazon — APPROVED** [review]

**Comment by peterzhuamazon** [comment]
Thanks, merged now.

**Pull request merged**
